platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -19,9 +19,11 @@ declare -r EXIT_SUCCESS="0"
 declare -r EXIT_FAILURE="1"
 
 declare -r QUERY_CMD="mlxfwmanager --query"
+declare -r LIST_CONTENT_CMD="mlxfwmanager -l"
```

Comment:
@ayurkiv-nvda suggest to use `--list-content` here

Reply:
Done